core/src/javascript/registrations/externalProvider/registerRoutes.js

```diff
@@ -0,0 +1,21 @@
+import {registry} from '@jahia/ui-extender';
+
+export const registerRoutes = function (t) {
+    const level = 'server';
+    const parentTarget = 'administration-server';
+
+    const path = '/administration/external-provider';
+    const route = 'external-provider';
+    registry.addOrReplace('adminRoute', `${level}-${path.toLowerCase()}`, {
```

Comment:
I would rather use add() unless there's a good reason to think we need to replace something

Reply:
I'll change it to add(), I had an issue with component rerendering and because the registration is in the function it was trying to readd it. I think callback mechanism may fix that.